core/src/com/po/fuck/Constants.java

```diff
@@ -18,4 +18,6 @@ public final class Constants {
 
     /* health bar */
     public static float HEALTHBAR_OFFSET = 5;
+    public static final int ENEMY_TAG = 0;
+    public static final int PLAYER_TEAM_TAG = 1;
```

Comment:
Not consistent naming

Reply:
Probably, it concept of "teams" can be outdated in the future, as, for example, neutral entities come into the game. Probably it could be replaced with "group" tags?

Reply:
If there will be some neutral entities they can have some `NEUTRAL_ENTITY_TAG`, I do not understand your argument

Reply:
If enemies will attack them, then it will be an interesting mechanic